src/components/screens/wallet/delegateProfile/delegateProfile.js

```diff
@@ -39,7 +40,7 @@ const DelegateProfile = ({
         />
         <PerformanceView
           t={t}
-          data={data.dpos.delegate}
+          data={userData}
```

Comment:
I prefer to pass the info in 2 props `delegateData` and `address` instead of creating a copy of it and pass it all together. Or either decouple the props or to pass the whole data component.

Reply:
The address is no longer needed so I've reverted the changes here